src/core/main.c

```diff
@@ -83,6 +83,11 @@
 #include "virt.h"
 #include "watchdog.h"
 
+#if HAS_FEATURE_ADDRESS_SANITIZER
+#include <sanitizer/lsan_interface.h>
+#endif
+
+
```

Comment:
duplicated empty line.

Reply:
Thank you! I'm not sure how I missed that. It's fixed now.